lib/bokken/types/pt_mobile.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Bokken.Types.Ecto.PtMobile do
```

Comment:
In our application logic we try to follow folder name base rules. For example, each module name has it own folder.

What do you think in changing this to `Bokken.Ecto.PtMobile` and make it `lib/bokken/ecto/pt_mobile.ex`?

Reply:
Yes I agree. I wasn't sure of where to put this module